lib/pages/fake_call_support.dart

```diff
@@ -0,0 +1,243 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:wakelock/wakelock.dart';
+
+import 'fake_incoming_call_screen.dart';
+
+class InstructionScreen extends StatefulWidget {
+  static final String route = '/fakeCallSupport';
+
+  @override
+  _InstructionScreenState createState() => _InstructionScreenState();
+}
+
+class _InstructionScreenState extends State<InstructionScreen> {
+  final GlobalKey<ScaffoldState> _scaffoldKey = GlobalKey<ScaffoldState>();
+
+  bool _activateBtnStatus;
+
+  static const values = <int>[0, 1, 2, 3];
+  static const textValues = <String>["Now", "1 min", "3 min", "5 min"];
+  int selectedValue = values.first;
+
+  final selectedColor = Colors.green;
+  final unselectedColor = Colors.white;
+
+  TextEditingController _fakeName;
+
+  @override
+  void initState() {
+    // TODO: implement initState
+    super.initState();
+    _fakeName = TextEditingController();
+    _activateBtnStatus = true;
+  }
+
+  @override
+  void dispose() {
+    // TODO: implement dispose
+    super.dispose();
```

Comment:
implement all dispose functionalities before super.dispose(), there should be no code written below super.dispose